web/src/app/schedules/ScheduleRuleForm.js

```diff
@@ -103,45 +103,16 @@ const query = gql`
 
 @withStyles(styles)
 @connect((state) => ({ zone: urlParamSelector(state)('tz', 'local') }))
```

Comment:
We have a custom hook to handle this url param selector, you can import it as:
`import { useURLParam } from '../actions/hooks'`
and set the param as:
`const [zone] = useURLParam('tz', 'local') `
Exmaple here: https://github.com/target/goalert/blob/master/web/src/app/schedules/ScheduleShiftList.js#L68